ts/Series/Pie/PieSeries.ts

```diff
@@ -576,6 +576,26 @@ class PieSeries extends Series {
          * @apioption plotOptions.pie.startAngle
          */
 
+        /**
+         *
+         *
+         *
+         *
+         * `thickness` 
+         *
+         *
+         *
+         * 
+         *
+         * 
```

Comment:
.
.
.
.
.
why?
.
.
.
.
.

but seriously, this should look more like `startAngle` (one option above).